ci/builder/requirements.txt

```diff
@@ -41,7 +41,7 @@ psycopg==3.2.5
 psycopg-binary==3.2.5
 # pydantic >= 2.9 causes pdoc 15.0.1 to fail: UserWarning: Error parsing type annotation dict[str, Any] | None for pydantic.main.BaseModel.__pydantic_extra__: 'function' object is not subscriptable
 # See also https://github.com/mitmproxy/pdoc/issues/741
-pydantic==2.10.6
+pydantic==2.8.2
```

Comment:
Unrelated?

Reply:
Had to cherry-pick that from another PR to make it green, but the other PR is already merged, so this is a no-op